### ISSUE TITLE
InputFile component has two onchange event handlers

### ISSUE THREAD

**Issue by pranavkm** [dotnet/aspnetcore#36573]
![image](https://user-images.githubusercontent.com/174281/133508891-802eb885-7acd-4968-9d01-2f578dc978c2.png)

The first is the vanilla `EventHandler<ChangeEventArgs>` which is not super useful in this scenario.

**Comment by NTaylorMullen** [comment]
@pranavkm is there a good way we can detect this scenario at tooling to do the right thing?

**Comment by pranavkm** [comment]
@TanayParikh said that the fact that `OnChange` shows when there is an `@` prefix is the problem. It's a vanilla property and should be filtered out since it has an `@`. I was originally cribbing about the fact that there are two confusingly similar named handlers on a type and we missed it in API review.

**Comment by NTaylorMullen** [comment]
Should we always remove `@onchange`?

**Comment by pranavkm** [comment]
`@onchange` is valid for most things, including `InputFile` since it maps the [HTML `change` event](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event). However, it's not polymorphic in that the `change` event for files includes additional data when coming from the browser, but Blazor's `ChangeEventArgs` does not have a way to expose that. 

tl;dr: Don't remove it, it's still very useful for all things not InputFile.

**Comment by NTaylorMullen** [comment]
> tl;dr: Don't remove it, it's still very useful for all things not InputFile.

Hmmm, so I'd hate to start adding special cases, is there a way to detect via the `TagHelperDescriptor`s to "not include" it?
